### PR TITLE
fix: regenerate cert for sendgrid oai

### PR DESCRIPTION
**Pull request by beebzz**

### CODE REVIEW

**Review by childish-sambino — COMMENTED** [review]
Were you able to verify locally this fixes the issue?

**Comment by beebzz** [comment]
> Were you able to verify locally this fixes the issue?

Doing this now, looks like there's some extra testing to do

File: prism/nginx/san.cnf [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+default_bits       = 2048
+distinguished_name = req_distinguished_name
+req_extensions     = req_ext
+[ req_distinguished_name ]
+countryName                 = Country Name (2 letter code)
+stateOrProvinceName         = State or Province Name (full name)
+localityName               = Locality Name (eg, city)
+organizationName           = Organization Name (eg, company)
+organizationalUnitName           = Organizational Unit Name (eg, team)
+commonName                 = Common Name (e.g. server FQDN or YOUR name)
```
Comment (childish-sambino) [inline prism/nginx/san.cnf:11]: Should remove these or add proper-ish values.

Comment (beebzz) [reply prism/nginx/san.cnf:11]: The `req_distinguished_name` field is just a template I believe; you're prompted for those values when you regenerate the cert

Comment (beebzz) [reply prism/nginx/san.cnf:11]: Ah never mind, looks like you can disable prompting and just default the values.

**Review by childish-sambino — APPROVED** [review]

**Pull request merged**